src/Common/src/Interop/Unix/System.Security.Cryptography.Native/Interop.RAND.cs

```diff
@@ -9,16 +9,15 @@ internal static partial class Interop
 {
     internal static partial class Crypto
     {
-        internal static unsafe bool GetRandomBytes(byte* pbBuffer, int count)
+        internal static bool GetRandomBytes(ref byte pbBuffer, int count)
         {
-            Debug.Assert(pbBuffer != null);
```

Comment:
Interesting.  I thought the thing that sparked this was a suggestion that `ref byte` instead of `byte*` could eliminate an `unsafe` from a place passing `null`.  Maybe it was just a general suggestion and I misapplied it :smile:.